src/cgame/etj_speed_drawable.h

```diff
@@ -2,11 +2,19 @@
 #include "etj_irenderable.h"
 #include "cg_local.h"
 #include <string>
+#include <list>
 
 namespace ETJump
 {
 	class DisplaySpeed : public IRenderable
 	{
+		struct StoredSpeed
+		{
+			int time;
+			float speed;
+		};
+
+		std::list<StoredSpeed> _storedSpeeds;
```

Comment:
Is there a reason to use list instead of a vector?

Reply:
Not really, it was just the first data structure that came to mind when I wanted something that is good at adding elements to the end and removing from the beginning and random access was not necessary.

Reply:
boost has circular buffer etc. for this use. Let's go with this for now.